definitions/npm/redux-saga_v0.11.x/flow_>=v0.28.x/redux-saga_v0.11.x.js

```diff
@@ -0,0 +1,482 @@
+/* @flow */
+
+/* eslint-disable */
+
+/**
+ * Shared interfaces between the modules 'redux-saga' and
+ * 'redux-saga/effects'
+ */
+declare interface $npm$ReduxSaga$Channel {
+  take: (cb: (msg: mixed) => void) => void,
+  close: Function,
+}
+
+declare type $npm$ReduxSaga$Task = {
+  '@@redux-saga/TASK': true,
+  isRunning: () => boolean;
+  isCancelled: () => boolean;
+  result: () => ?mixed;
+  error: () => ?Error;
+  cancel: () => void;
+  done?: Promise<*>;
+}
+
+declare interface $npm$ReduxSaga$Buffer {
+  isEmpty(): boolean;
+  put(msg: any): void;
+  take(): any; 
+}
+
+declare type $npm$ReduxSaga$IOEffect = {
+  '@@redux-saga/IO': true,
+}
+
+declare module 'redux-saga' {
+  // Parts for the Channel interface
+  declare type EmitterFn = (msg: any) => void;
+  declare type UnsubscribeFn = () => void;
+  declare type SubscribeFn = (emitter: EmitterFn) => UnsubscribeFn;
+  declare type MatcherFn = (msg: any) => boolean;
+
+  declare type Pattern = string | Array<string> | (action: Object) => boolean;
+
+  declare type IOEffect = $npm$ReduxSaga$IOEffect;
+  declare export type Channel = $npm$ReduxSaga$Channel;
+  declare export type Buffer = $npm$ReduxSaga$Buffer;
+  declare export type Task = $npm$ReduxSaga$Task;
+
+  declare export interface SagaMonitor {
+    effectTriggered(options: {
+      effectId: number,
+      parentEffectId: number,
+      label: string,
+      effect: Object,
+    }): void;
+    effectResolved(effectId: number, result: any): void;
+    effectRejected(effectId: number, err: Error): void;
```

Comment:
I think `err` should be `any` (will be typically an `Error` but it's not enforced just like the Promise rejection value which can be `any` type)